### PR TITLE
Add livenessProbe and readinessProbe configuration

### PR DESCRIPTION
**Pull request by misternobody**
Add the customisation of liveness and readiness probes.

### CODE REVIEW

**Comment by weeco** [comment]
Hey @misternobody ,
thank you for submitting this PR. Could you define the default values as part of the `values.yaml` rather than using the `| default xy` defaults? This way the fields are better documented in the values.yaml - this seems to be way more comment among the Helm community. Thanks!

**Comment by misternobody** [comment]
Hi @weeco 
Make sense, fixed!

**Comment by misternobody** [comment]
Hey @weeco 
Please review when you have a min.

File: README.md [hunk 1]
```diff
@@ -68,6 +68,16 @@ helm install -f values.yaml kowl cloudhut/kowl
...
+| `livenessProbe.initialDelaySeconds` | Number of seconds after the container has started before liveness probe is initiated | 0 |
+| `livenessProbe.periodSeconds` | How often (in seconds) to perform the probe | 10 |
+| `livenessProbe.timeoutSeconds` | Number of seconds after which the probe times out | 1 |
+| `livenessProbe.successThreshold` | Minimum consecutive successes for the probe to be considered successful after having failed | 1 |
+| `livenessProbe.failureThreshold` | When a probe fails, Kubernetes will try failureThreshold times before giving up | 3 |
+| `readinessProbe.initialDelaySeconds` | Number of seconds after the container has started before readiness probe is initiated | 10 |
+| `readinessProbe.periodSeconds` | How often (in seconds) to perform the probe | 10 |
+| `readinessProbe.timeoutSeconds` | Number of seconds after which the probe times out | 1 |
+| `readinessProbe.successThreshold` | Minimum consecutive successes for the probe to be considered successful after having failed | 1 |
+| `readinessProbe.failureThreshold` | When a probe fails, Kubernetes will try failureThreshold times before giving up | 3 |
```
Comment (weeco) [inline README.md:80]: Can you put ticks around the default values in that table so that it's consistent with the rest?

Comment (misternobody) [reply README.md]: Fixed

**Review by weeco — APPROVED** [review]
Thanks!

**Pull request merged**